docs/pages/getting-data-with-pelican/client.mdx

```diff
@@ -241,6 +241,91 @@ The Pelican client supports a variety of command line flags that modify the clie
 - **-r or --recursive:** Takes no argument and indicates to Pelican that all sub paths at the level of the provided namespace should be copied recursively. This option is only supported if the origin supports the WebDav protocol.
 - **-t or --token:** Takes a path to a file containing a signed JWT, and is used to download protected objects.
 
+## Listing Objects on an Origin
+
+To use the pelican client to list objects in a collection, use Pelican's `object ls` sub-command
+
+```bash
+pelican object ls pelican://<federation-url></namespace-prefix></path/to/collection>
+```
+
+This will only be successful when listings are enabled in the Pelican origin. To learn more about origin capabilities, go to the [Origin docs](../federating-your-data/origin.mdx).
+
+If listings are not enabled, you will see the following error:
+
+```
+ERROR[2025-02-07T15:27:05-06:00] Failure getting pelican://osdf-director.osg-htc.org:443/nrao-ardg: failed to perform list request: Collections URL not found in director response. Are you sure there's an origin for prefix /<namespace-prefix> that supports listings?
+```
+
+## Getting Object Data From a Federation with Stat
```

Comment:
Same comment about documenting when tokens are needed and how to provide one, or otherwise linking to documentation that explains it.

Also, I'd remove "From a Federation". I think it's enough to say "Getting Object Data with Stat".